specs/disciplina/witness/api/explorer.yaml

```diff
@@ -0,0 +1,215 @@
+swagger: "2.0"
+info:
+  description: "This is an HTTP API for interacting with Disciplina Witness node. It is used by Block Explorer frontend to request info about blocks, addresses and transactions."
+  version: 0.1.0
+  title: Disciplina Block Explorer API
+  termsOfService: "https://disciplina.io/tnc.pdf"
+  contact:
+    email: hi@serokell.io
+  license:
+    name: Apache 2.0
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "localhost:8091"
+basePath: /api/witness/v1
+schemes:
+  - https
+  - http
+paths:
+  /blocks:
+    get:
+      summary: >-
+        Get list of blocks.
+        Only the batch of N most recent blocks are returned at once.
```

Comment:
Of how many?

Reply:
Please choose an arbitrary number.

Reply:
Let it be a parameter maybe?

Reply:
We still need to have some default amount or hard limit, as we probably don't want to serve the entire blockchain via this API

Reply:
Yes, let's say that the limit is 100, but let it be a parameter anyway.